api/routers/http/routes.go

```diff
@@ -17,6 +21,161 @@ func AddRoutes(r *gin.Engine, authMiddleware *jwt.GinJWTMiddleware) *gin.RouterG
 			})
 		})
 
+		api.GET("/users", func(c *gin.Context) {
+			// Create a list of random users
+			users := controllers.GetUsers()
+			c.JSON(200, gin.H{
+				"data": users,
+			})
+		})
+
+		api.GET("/users/:id", func(c *gin.Context) {
+			// Get the id parameter from the URL
```

Comment:
Ideally we have as little logic in the router handlers, so it stays clear what HTTP methods we do have available. Would expect something like this:


    r.GET("/videowalls/:id", func(c *gin.Context) {
        middleware.WithUserMetadata(c, log, controllers.GetVideowall)
    })